constellation_forms/static/constellation_forms/css/forms.css

```diff
@@ -0,0 +1,103 @@
+.small-card {
+    margin-bottom: 10px;
+    width: 100%;
+}
+
+.widget-list {
+    margin: 0;
+}
+
+.widget-list .mdl-list__item {
+    min-height: 30px;
+    padding: 0 16px;
+}
+
+.form-options .mdl-switch {
+    margin-bottom: 10px;
+}
+
+.huge-textfield .mdl-textfield__input {
+    font-size: 40px;
+}
+
+.huge-textfield .mdl-textfield__label {
+    font-size: 40px;
+    line-height: 40px;
+    top: 31px;
+}
+
+.form-part {
+    border-bottom: 1px solid #dddddd;
+    padding: 16px;
+    position: relative;
+}
+
+/* Fast transition for box-shadow */
+.form-part::after {
+    box-shadow: 0 2px 5px 0 rgba(0,0,0,.14),0 1px 10px 0 rgba(0,0,0,.12),0 2px 4px -1px rgba(0,0,0,.2);
+    content: "";
+    height: 100%;
+    margin: -16px;
+    opacity: 0;
+    position: absolute;
```

Comment:
The shadow on my title and description form-part is far to the right.
![form_title](https://cloud.githubusercontent.com/assets/2491132/23540540/797b2eb6-ffa7-11e6-8003-a7b43869ae8d.png)